go/storage/fuzz/fuzz.go

```diff
@@ -0,0 +1,53 @@
+// +build gofuzz
+
+package fuzz
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature/signers/file"
+	commonFuzz "github.com/oasislabs/oasis-core/go/common/fuzz"
+	"github.com/oasislabs/oasis-core/go/common/identity"
+	"github.com/oasislabs/oasis-core/go/storage"
+)
+
+const (
+	dataDir string = "/tmp/oasis-node-fuzz-storage"
+	identityDir string = dataDir + "/identity"
+)
+
+func Fuzz(data []byte) int {
+	signerFactory := file.NewFactory(identityDir, signature.SignerNode, signature.SignerP2P, signature.SignerConsensus)
+	identity, err := identity.Load(identityDir, signerFactory)
+	if err != nil {
+		panic(err)
+	}
+
+	// Every fuzz worker should get its own database, otherwise we can't work.
+	localDB, err := ioutil.TempDir(dataDir, "worker")
```

Comment:
How is this shared between invocations? Won't each `Fuzz` invocation start with its own database?

Reply:
Yep, comment was mildly wrong.